### PR TITLE
Adding the main function file and the logic files of the GUI

### PR DESCRIPTION
**Pull request by wenyifeng1123**
Adding two logic files of the GUI, and adding the main function file(matplotlibwidget.py) of the GUI. Besides, fixing the visualization functions in file network_visualization.py

### CODE REVIEW

File: deepvis/matplotlibwidget.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+    
+
+
+
+
+
+
+class MyMplCanvas(FigureCanvas):
+    """FigureCanvas的最终的父类其实是QWidget。"""
```
Comment (thomaskuestner) [inline deepvis/matplotlibwidget.py]: please only unicode characters

File: deepvis/matplotlibwidget.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+    def __init__(self, parent=None, width=15, height=15):
+        # 配置中文显示
```
Comment (thomaskuestner) [inline deepvis/matplotlibwidget.py]: please translate to english

File: deepvis/matplotlibwidget.py [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+        plt.rcParams['font.family'] = ['SimHei']  # 用来正常显示中文标签
+        plt.rcParams['axes.unicode_minus'] = False  # 用来正常显示负号
+
+        self.fig = plt.figure(figsize=(width, height))  # 新建一个figure
+
+        FigureCanvas.__init__(self, self.fig)
+        self.setParent(parent)
+
+        '''定义FigureCanvas的尺寸策略，这部分的意思是设置FigureCanvas，使之尽可能的向外填充空间。'''
```
Comment (thomaskuestner) [inline deepvis/matplotlibwidget.py]: no chinese characters

File: deepvis/matplotlibwidget.py [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+        FigureCanvas.setSizePolicy(self,
+                                   QSizePolicy.Expanding,
+                                   QSizePolicy.Expanding)
+        FigureCanvas.updateGeometry(self)
+
+    '''绘制静态图，可以在这里定义自己的绘图逻辑'''
```
Comment (thomaskuestner) [inline deepvis/matplotlibwidget.py]: same here

**Comment by wenyifeng1123** [comment]
All fixed

**Pull request merged**